httpd/handler.go

```diff
@@ -164,7 +164,7 @@ func (h *Handler) serveWrite(w http.ResponseWriter, r *http.Request, user *influ
 	for {
 		if err := dec.Decode(&br); err != nil {
 			if err.Error() == "EOF" {
-				w.WriteHeader(http.StatusOK)
+				w.WriteHeader(http.StatusCreated)
 				return
```

Comment:
In the event a datapoint exists already, our system might overwrite. Will we have a different code for that? I really think the most natural thing for this API to return is 200, and is what client library developers will expect.